test/postControllerTest.js

```diff
@@ -0,0 +1,96 @@
+/* global it, describe */
+const express = require('express');
+const request = require('supertest');
+const postController = require('../controllers/postController');
+const sinon = require('sinon');
+const postService = require('../services/postService');
+
+const app = express();
+
+app.set('view engine', 'ejs');
+app.use(express.static('../public'));
+
+app.use('', postController);
+const server = app.listen(3000);
```

Comment:
looks like you don't even need to bind when using supertest

Reply:
Oh wow, nice.